samples/get_operation_status.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  operationFullId = 'projects/[projectId]/locations/us-central1/operations/[operationId]'
+) {
+  // [START automl_get_operation_status]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const operationFullId = 'projects/[projectId]/locations/us-central1/operations/[operationId]';
+
```

Comment:
construct the `operationFullId` here it'll be something like

operationFullId = `projects/${projectId}/locations/${location}/operations/${operationId}